### DISCUSSION TITLE
Treeconomy Ackron Mixed Afforestation (WCC 103000000004596)

### DISCUSSION THREAD

**Discussion by hai-yr** [opentaps/open-climate-investing#38] [Offsets]
"Ackron Mixed is one of the oldest projects in the UK Woodland Carbon Code, located near Melvich in the north of Scotland. The land, previously used for sheep grazing, has been replanted with a broadleaf and conifer mix. Planting was intentionally spaced to allow UK priority species to thrive, including black grouse and short-eared owls.

This project [offers] ... nature-based carbon removal credits, which have not been pre-sold and which have gone through their 15-year verification.

Note: Credits certified by the Woodland Carbon Code can only be used by UK-registered companies to claim compensation for emissions while complying with WCC regulations."
Vintage: 2021

https://mer.markit.com/br-reg/public/project.jsp?project_id=103000000004596

**Comment by sichen1234** [comment]
Is there a place to buy this credit?

**Reply by hai-yr** [reply]
266 tonnes available on Patch.io: https://dashboard.patch.io/projects/pro_prod_aff3d209937bb783988f028bc2c9e562

**Comment by hai-yr** [comment]
I will be adding ratings under here for the same criteria as BeZero plus a couple of added criteria:

Additionality/Baseline X/5
Commentary:
Financial attractiveness -> 
Baseline area -> 

Permanence/risk of reversal: X/5
Commentary:

Leakage X/5
Commentary:

Incentives X/5
Commentary: 

Over-crediting X/5
Commentary:

Political environment X/5
Commentary:

Transparency/availability of data X/5
Commentary: 

Transition to net zero 3/5
Commentary: As an afforestation/tree planting project, the project qualifies as carbon removal vs. purely emissions avoidance, albeit with an inherent risk of impermanence. The Oxford Principles for Offsetting suggest that these types of removals should be replaced over time with more permanent means of storing carbon.
